code-challenges.js

```diff
@@ -12,23 +12,79 @@
 var temp = 42
 // var temp = 350
 // var temp = 212
-
+// var temp = -460
+// var temp = "fish"
+
+// if (temp === 212) {
+//     console.log("212 is at boiling point")}
+// else if(temp < -459.67) {
+//     console.log("Impossible, value is below absolute zero")}
+// else if (temp < 212) {
+//     console.log(`${temp} is below boiling point`)}
+// else if (temp > 212) {
+//     console.log(`${temp} is above boiling point`)} 
+// else {
+//     console.log("Invalid, please input a number datatype")}
```

Comment:
Haha nice edge cases. In the future, please comment IN the code you are submitting.